kai/reactive_codeplanner/task_runner/analyzer_lsp/validator.py

```diff
@@ -156,7 +156,7 @@ def __parse_analyzer_lsp_output(
                         class_to_use = AnalyzerDependencyRuleViolation
                     validation_errors.append(
                         class_to_use(
-                            file=urlparse(i.uri).path,
+                            file=urlparse(i.uri).path.lstrip("/"),
```

Comment:
Why this change?

Reply:
I was getting relative paths starting with a slash (ie, `/example/coolstore/src/...`), I just had this in to fix it though I did mean to follow up on it. Can remove from this PR, also curious if anyone else is seeing that

Reply:
we should write an issue in analyzer and track that down

Reply:
Should I drop it here? Nothing works for me without this change

Reply:
No we should keep it, but I just want to create a follow on issue because IIRC this was a conversation with the IDE/UX folks and I want to make sure that we track it. 

I am good with keeping just wanted to know :)